src/components/Modal/Modal.scss

```diff
@@ -11,4 +11,10 @@
   justify-content: center;
   overflow-y: auto;
   -webkit-overflow-scrolling: touch;
+
+  &--overlay {
+    align-items: baseline;
+    background: rgba(255, 255, 255, 0.9);
+    z-index: z(9990);
```

Comment:
I generally don't think including z-indexes in a lib like this is a good idea since it presupposes a lot about the application consuming it (like what level the persistent nav is sitting at, etc).

Reply:
Yeah I agree. Do you think we just need to have a base style in the app for `.Modal` that sets the z index?

Reply:
Yeah, that's a good idea 💚